src/sparsevector.jl

```diff
@@ -1486,6 +1496,15 @@ function Base._mapreduce(f, op, ::IndexCartesian, A::SparseVectorUnion{T}) where
     end
     _mapreducezeros(f, op, T, rest, ini)
 end
+Base._mapreduce(f, op, ::Base.IndexCartesian, A::Adjoint{<:Any,<:SparseVectorUnion}) =
+    Base._mapreduce(f∘adjoint, op, IndexCartesian(), parent(A))
+Base._mapreduce(f, op, ::Base.IndexCartesian, A::Transpose{<:Any,<:SparseVectorUnion}) =
```

Comment:
i really think this should go in LinearAlgebra for all types.

Reply:
Yes. Can you take it there? I'll remove it here.

Reply:
For the record, I removed special versions of the following generic ones to be added:

```julia
Base._mapreduce(f, op, ::Base.IndexCartesian, A::Adjoint) =
    Base._mapreduce(f∘adjoint, op, IndexCartesian(), parent(A))
Base._mapreduce(f, op, ::Base.IndexCartesian, A::Transpose) =
    Base._mapreduce(f∘transpose, op, IndexCartesian(), parent(A))
Base._count(f, x::Adjoint, ::Colon, init) = Base._count(f∘adjoint, parent(x), :, init)
Base._count(f, x::Transpose, ::Colon, init) = Base._count(f∘transpose, parent(x), :, init)
```